Sources/MapboxMaps/Offline/OfflineErrors.swift

```diff
@@ -37,20 +40,21 @@ public enum TileRegionError: LocalizedError, CoreErrorRepresentable, Equatable {
             self = .diskFull(message)
         case .other:
             self = .other(message)
+        case .tileCountExceeded:
+            self = .tileCountExceeded(message)
+        @unknown default:
```

Comment:
Is this something we need to do for any other enums in the SDK?

Reply:
I added this because of a compiler warning.

Reply:
There were only 2 instances and I fixed them both.